Remove key definition in the package loading.

sstepashka opened the pull request:
Avoid modification of Emacs behavior on the package loading.

In order to follow [the coding convention](https://www.gnu.org/software/emacs/manual/html_node/elisp/Coding-Conventions.html), disable key definition in the package loading

cslux commented:
Hey @sstepashka! Good point. We should however also update the README.org file which mentions it in several places that `org-journal-new-entry` can be called with the key `C-c C-j`.

sstepashka commented:
Hey @cslux! I updated the document and added "C-c C-j" in `org-journal-mode`. It was mentioned in the documentation, but seems like it wasn't defined in there.

bastibe commented on org-journal.el:
```diff
@@ -399,7 +400,8 @@ This runs once per date, before `org-journal-after-entry-create-hook'.")
 ;; Key bindings
 (when (and (stringp org-journal-prefix-key) (not (string-empty-p org-journal-prefix-key)))
-  (let ((command-table '(("f" . org-journal-next-entry)
+  (let ((command-table '(("j" . org-journal-new-entry)
```
This now contains "j" twice.

sstepashka replied:
Fixed.

bastibe commented:
As this is a breaking change, it should probably get a (temporary) mention near the top of the README.

Otherwise, looks good to me!
